### PR TITLE
nut: new plugin to monitor Network UPS Tools (NUT)

### PR DESCRIPTION
**Pull request by kimheino**

### CODE REVIEW

File: plugins/nut/nut [hunk 1]
```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/python3 -tt
```
Comment (sumpfralle) [inline plugins/nut/nut:1]: Please use `/usr/bin/env python3` without arguments.

File: plugins/nut/nut [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+# -*- coding: utf-8 -*-
+
+"""Munin plugin to monitor Network UPS Tools (NUT)
```
Comment (sumpfralle) [inline plugins/nut/nut:4]: Please apply `perlpod`.

**Review by sumpfralle — COMMENTED** [review]
It looks good!

Do you think, that this could supersede the already existing plugin `power/nut`?
In this case, you could remove the other one.
If not, then it would be nice, if you could rename the plugin. For now `munin-get` and the plugin gallery do not really support plugins with the same name properly ...

**Comment by kimheino** [comment]
I somehow missed already existing plugins in "power" subdir and therefore wrote my own. I'll close this pull request as existing nutups2_ has almost same features.

See pull request #1199 instead to improve nutups2_.

**Pull request closed without merging**